packages/core-services/src/lib/asyncLocalStorage.ts

```diff
@@ -0,0 +1,5 @@
+import type { IServiceContext } from '../types/ServiceClass';
+import { FibersContextStore } from './ContextStore';
+
+// TODO Evalute again using AsyncContextStore instead of FibersContextStore in a future Meteor release (after 2.5)
```

Comment:
Aren't we in meteor 2.6? :trollface:

Reply:
true.. not sure why Meteor 2.5 though, the bump to Node 14 was on Meteor 2.3 .. I'd say we should re-evaluate this when Meteor updates to Node 16, that's when Fibers will be removed from Meteor.

Reply:
Noice